service/src/main/java/uk/gov/hmcts/reform/fpl/model/AllocatedJudge.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class AllocatedJudge {
```

Comment:
do we need the new java model?

Reply:
We need this new java model as we check that this AllocatedJudge is present before we allow the user to seal the SDO

Reply:
hmm, just concerns me being so similar to judgeAndLegalAdvisor type 😕

Reply:
discussed and changed to Judge